Apka Kursy/Controllers/AccountController.cs

```diff
@@ -10,19 +11,23 @@ namespace Apka_Kursy.Controllers
     public class AccountController : ControllerBase
     {
         private readonly IAccountService _accountService;
-        public AccountController(IAccountService accountService) 
+
+        public AccountController(IAccountService accountService)
         {
             _accountService = accountService;
         }
+
         [HttpPost("register")]
-        public ActionResult RegisterUser([FromBody] RegisterUserDto dto)//Akcja rejestracji
+        [AllowAnonymous]
+        public ActionResult RegisterUser([FromBody] RegisterUserDto dto)
         {
             _accountService.RegisterUser(dto);
             return Ok();
         }
 
         [HttpPost("login")]
-        public ActionResult Login([FromBody] LoginDto dto)//Akcja logowania
+        [AllowAnonymous]
```

Comment:
To jest niepotrzebne bo domyślnie jest ustawione ze kazdy moze dostac sie do endpointu jak nie ma autohrization.